vendor/engines/secure_rooms/spec/features/view_facility_occupancies_spec.rb

```diff
@@ -0,0 +1,66 @@
+require "rails_helper"
+
+RSpec.describe "Viewing Occupancies" do
+  let(:facility) { create(:setup_facility) }
+  let(:secure_room) { create(:secure_room, facility: facility) }
+  let(:facility_staff) { create(:user, :staff, facility: facility) }
+  before { login_as facility_staff }
+
+  context "with no in-progress occupancies" do
+    it "shows no occupancies" do
+      visit facility_occupancies_path(facility)
+
+      expect(current_path).to eq(facility_occupancies_path(facility))
+      expect(page).to have_content('There are no "In Process" occupancies.')
+    end
+  end
+
+  context "with occupancy" do
+    let(:account) { create :account, :with_account_owner, owner: facility_staff }
+
+    before do
+      # Allow purchase of order
+      allow_any_instance_of(Order).to receive(:cart_valid?).and_return(true)
```

Comment:
My new PR #1024 has some new factories that should allow for avoiding these kinds of stubbing. We can clean them up later, though. Especially in feature specs, I shy away from these kind of stubs.